index.html

```diff
@@ -41,6 +41,191 @@ <h1 class="primary-heading"> I’m Mariane Glad to see you! </h1>
 
         </ul>
     </section>
+
+    <section class="Projects">
+        <section class="Card-Works">
+            <header class="Snapshot">
+                <img src="./img/Snapshoot Portfolio.png" alt="snapshot poortfolio" class="snapshot">
+
+            </header>
+            <div class="info-block">
+                <h1 class="project-title">Tonic</h1>
+                <ul class="frame">
+                    <span class="client">
+                        CANOPY</span>
+                    <img src="./img/Counter.png" alt="" class="counter">
+                    <span class="role">Back End Dev</span>
+                    <img src="./img/Counter.png" alt="" class="counter">
+                    <span class="year">2015</span>
+    
+                </ul>
+                <span class="details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </span>
+                <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+            
+                    <button class="see-project">See Project</button>
+        
+            </div>
+        </section>
+
+        <section class="Card-Works">
+            <header class="Snapshot">
```

Comment:
I suggest you remove this header element here. Header element are mostly used for navigation and your page should follow a `header -> main -> footer` theme. Having this header element at the center of your app confuses screen readers. To learn more [read](https://www.w3schools.com/tags/tag_header.asp)